<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>705</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>705</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculate your way!</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-calculator">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0.867, y2:1, stop:0 rgba(255, 85, 255, 152), stop:0.466667 rgba(85, 255, 255, 205), stop:0.577778 rgba(85, 255, 255, 223), stop:1 rgba(0, 255, 0, 135));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>220</y>
      <width>135</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="frf"/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 47, 48);
color: rgb(255, 255, 255);
border:1px solid;
border-bottom-left-radius: 3px;
border-bottom-right-radius: 3px;
</string>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>135</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border: 2px solid;
border-bottom: 1px solid;
border-top-left-radius: 3px;
border-top-right-radius: 3px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../course work/img/V1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>295</y>
      <width>135</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border: 2px solid;
border-bottom: 1px solid;
border-top-left-radius: 3px;
border-top-right-radius: 3px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../course work/img/N97ByE0WfPwRXceH3tk-TnxkkDQB1HTtKpBeoB7OVGCn4L0CapHRVrpoOVNRnu94bgD34JLV3EuCw-xblu8H3M3d.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>135</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border: 2px solid;
border-bottom: 1px solid;
border-top-left-radius: 3px;
border-top-right-radius: 3px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../course work/img/V2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>415</y>
      <width>135</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 47, 48);
border:1px solid;
border-bottom-left-radius: 3px;
border-bottom-right-radius: 3px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string> Overall speed: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>135</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="frf"/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 47, 48);
color: rgb(255, 255, 255);
border:1px solid;
border-bottom-left-radius: 3px;
border-bottom-right-radius: 3px;
</string>
    </property>
    <property name="text">
     <string>First ship speed</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>135</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="frf"/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 47, 48);
border:1px solid;
border-bottom-left-radius: 3px;
border-bottom-right-radius: 3px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Second  ship speed</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>415</y>
      <width>135</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 47, 48);
border:1px solid;
border-bottom-left-radius: 3px;
border-bottom-right-radius: 3px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string> Distance: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>100</y>
      <width>135</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border: 2px solid;
border-bottom: 1px solid;
border-top-left-radius: 3px;
border-top-right-radius: 3px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../course work/img/time.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>295</y>
      <width>135</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border: 2px solid;
border-bottom: 1px solid;
border-top-left-radius: 3px;
border-top-right-radius: 3px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../course work/img/Vr.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>310</y>
      <width>135</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border: 2px solid black;
border-radius: 3px;
background-color: rgb(0, 118, 254);</string>
    </property>
    <property name="text">
     <string>Calculate!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>370</y>
      <width>135</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border: 2px solid black;
border-radius: 3px;
background-color: rgb(0, 118, 254);</string>
    </property>
    <property name="text">
     <string>How to use it?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
